libsession/src/main/java/org/session/libsession/messaging/utilities/SodiumUtilities.kt

```diff
@@ -3,27 +3,31 @@ package org.session.libsession.messaging.utilities
 import com.goterl.lazysodium.LazySodiumAndroid
 import com.goterl.lazysodium.SodiumAndroid
 import com.goterl.lazysodium.interfaces.GenericHash
-import com.goterl.lazysodium.interfaces.Sign
 import com.goterl.lazysodium.utils.Key
 import com.goterl.lazysodium.utils.KeyPair
 import org.session.libsignal.utilities.Hex
 import org.session.libsignal.utilities.toHexString
 import kotlin.experimental.inv
 
 object SodiumUtilities {
-
     private val sodium by lazy { LazySodiumAndroid(SodiumAndroid()) }
 
+    private const val scalarLength: Int = 32 // crypto_core_ed25519_scalarbytes
```

Comment:
should we standardise constant values to all caps with underscores? probably a decision that we could discuss and do consistently across new code

Reply:
Certainly, I'll update these, thanks